src/Pages/Channels.razor

```diff
@@ -280,6 +314,20 @@
         ToastService.ShowSuccess("Text copied");
     }
 
+    private double GetPercentageBalance(Channel channel)
+    {
+        try
+        {
+            var result = LightningService.GetChannelBalance(channel).Result;
```

Comment:
Any reason for not using async/await here?

Reply:
Nope, I just added it